DEFCON Z/Assets/Sounds/gameSound.cs

```diff
@@ -0,0 +1,24 @@
+﻿using System.Collections;
+using System.Collections.Generic;
+using UnityEngine;
+using DefconZ;
+
+namespace DefconZ
+{
+    public class GameSound : MonoBehaviour
```

Comment:
Please rename .cs file to match the class name.
Filename should really be capitalised like the class name.